bigquery_etl/glam/glean_scalar_aggregates.py

```diff
@@ -67,7 +67,12 @@ def generate_sql(
             DATE(submission_timestamp) AS submission_date,
             client_info.client_id,
             REPLACE(ping_info.ping_type, "_", "-") as ping_type,
-            SPLIT(client_info.app_display_version, '.')[OFFSET(0)] AS app_version,
+            COALESCE(
+                SAFE_CAST(
+                    SPLIT(client_info.app_display_version, '.')[OFFSET(0)]
+                    AS INT64
+                ),
+            0) AS app_version,
```

Comment:
There are pings with names like `Nightly 191010 06:01` that need to be filtered out. This logic is expecting semantic versioning. 
https://sql.telemetry.mozilla.org/queries/68262/source